PlanningPokerBackend.Tests/GamesTests.cs

```diff
@@ -36,5 +36,17 @@ public GamesTests()
             var response2 = await _client.PostAsync("/api/games/start", new StringContent(JsonConvert.SerializeObject(new TokenBody() { Token = token }), Encoding.UTF8, "application/json"));
             Assert.Equal(HttpStatusCode.OK, response2.StatusCode);
         }
+        [Fact]
+        public async Task IsStarted_Returns_True_If_Game_Is_Started()
+        {
+            LoginUser dave = new LoginUser() { Email = "davemurray@mail.com", Password = "password" };
```

Comment:
You use the same user for both tests, so you could extract the login data and store it as reaonly private parameter 😄

Reply:
@mzwierzchlewski good idea ;)